app/src/main/res/drawable/interaction_read_only_white_background.xml

```diff
@@ -4,7 +4,7 @@
   android:shape="rectangle">
   <stroke
     android:width="1dp"
-    android:color="@color/color_def_oppia_stroke_black"/>
+    android:color="@color/color_def_black_80"/>
```

Comment:
@rt4914 this drawable is not use anywhere. So what to do with this?

Reply:
Simply remove this. If any tests fail that means we have to add it back.

Reply:
> Simply remove this. If any tests fail that means we have to add it back.

Okay, No test is failing as for now.

Reply:
Okay I'll check and remove it if all good.